developer/tutorial/plugin-layouts-3.md

```diff
@@ -142,10 +180,17 @@ Reaction.registerPackage({
 ```
 
 You can see we specified several things there. The most important thing was the "layout" record, which refers to the new
-layout template we will create in the next chapter. We also specify which templates we want for the header and footer (we are just keeping the default for now),
+layout template we will create in the next chapter. We also specify which templates we want for the header and footer (we are just keeping the default for now, which
+are build-in React components called NavBar & Footer),
 and what's the main template that we render and that's `products`. We also
 specified which template we would use for a "notFound". When we get to the routing and template more of this will make sense.
 
+One important aspect is the casing of the properties within the `structure`.
+React component names start with capital letters, whereas Blaze templates are
+beginning with a lower character. For now it's not possible to use React
```

Comment:
should be "Blaze templates begin with a lowercase letter".

Reply:
React also?

Reply:
I mean "Blaze templates are beginning with a lower character" is grammatically incorrect. Should be "Blaze templates begin with a lowercase letter"